src/app/core/cache/builders/remote-data-build.service.ts

```diff
@@ -186,10 +186,10 @@ export class RemoteDataBuildService {
             rdArr.push(this.buildSingle(href, resourceConstructor));
           });
 
-          if (isList) {
-            links[relationship] = this.aggregate(rdArr);
-          } else if (rdArr.length === 1) {
+          if (rdArr.length === 1) {
```

Comment:
This undoes [this bugfix](https://github.com/DSpace/dspace-angular/pull/129/files), likely a merge conflict.